File: csharp/module1/solutions/solution2.cs
public double GetFinalPrice(double price) {
    const double taxRate = 0.15;
    const double discountRate = 0.1;
    double priceAfterDiscount = price - price * discountRate;
    double priceAfterTax = priceAfterDiscount + priceAfterDiscount * taxRate;
    return Math.Round(priceAfterTax, 2);
}

File: javascript/module1/solutions/solution2.js
function getFinalPrice(price) {
  const taxRate = 0.15;
  const discountRate = 0.1;
  const priceAfterDiscount = price - price * discountRate;
  const priceAfterTax = priceAfterDiscount + priceAfterDiscount * taxRate;
  return priceAfterTax.toFixed(2);
}
